CMakePresets.json

```diff
@@ -13,7 +13,7 @@
       "generator": "Ninja",
       "cacheVariables": {
         "MPI": "ON",
-        "TESTING": "OFF",
+        "TESTING": "ON",
```

Comment:
This default setting should be also replicated in the `CMakeLists.txt`

Reply:
OK